src/views/Blocks/HomeBlocks.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import { ControlledAppbar } from '../../components'
+import { ControlledButton } from '../../components'
```

Comment:
Remember you should import component if it will be used on the future ui design development.